packages/parser-javascript/README.md

```diff
@@ -40,10 +40,11 @@ This `parser` emits the following events:
 * `parse::end::javascript`, of type `ScriptParse` which contains the following
   information:
 
-* `resource`: the parsed resource. If the JavaScript is in
-  a `script tag` and not a file, the value will be `Internal
-  javascript`.
-* `sourceCode`: a `eslint` `SourceCode` object.
+  * `element`: an `IAsyncHTMLElement` reference if the source was inline in HTML; `null` otherwise.
```

Comment:
We are missing `ast` here from what I'm seeing down below. Not your fault though but can you please udpate it?

Reply:
Done!